apps/build.sh

```diff
@@ -106,7 +106,7 @@ for x in $IMAGES ; do
 	# allow the docker-build.conf to override our manifest platforms
 	MANIFEST_PLATFORMS="${MANIFEST_PLATFORMS-${MANIFEST_PLATFORMS_DEFAULT}}"
 
-	ct_base="hub.foundries.io/${FACTORY}/$x"
+	ct_base="${hub_fio}/${FACTORY}/$x"
```

Comment:
Where `hub_fio` is defined? In the login function. Maybe we should introduce some env. var for it.

Reply:
the docker-login function sets this as a variable